### DISCUSSION TITLE
No temperature on v3

### DISCUSSION THREAD

**Discussion by MaximeCheramy** [cdnninja/yoto_ha#113] [Q&A] [answered]
Hi,

Thanks a lot for your work, it's very appreciated. I'm using the last version of the integration and firmware is v2.18.4.

In the list of sensors, I don't see the temperature. Is there anything I can do to fix that?

![image](https://github.com/user-attachments/assets/c51288ed-afea-4bc0-9e8e-6f0437be32a7)

**Comment by cdnninja** [comment]
Did you have temp before?

**Comment by cdnninja** [comment, marked as answer]
Should be fixed now.

**Comment by MaximeCheramy** [comment]
I confirm that it works now. Thank you
